crates/anvil/tests/it/fork.rs

```diff
@@ -1473,6 +1473,51 @@ async fn test_reset_dev_account_nonce() {
     assert!(receipt.status());
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn test_reset_updates_cache_path_when_rpc_url_not_provided() {
+    let config: NodeConfig = fork_config();
+
+    let (mut api, _handle) = spawn(config).await;
+    let info = api.anvil_node_info().await.unwrap();
+    let number = info.fork_config.fork_block_number.unwrap();
+    assert_eq!(number, BLOCK_NUMBER);
+
+    async fn get_block_from_cache_path(api: &mut EthApi) -> u64 {
+        let db = api.backend.get_db().read().await;
+        let cache_debug = format!("{:?}", db.maybe_inner().unwrap().cache());
+        let re = regex::Regex::new(r#"JsonBlockCacheDB \{ cache_path: Some\("([^"]+)"\)"#).unwrap();
```

Comment:
can we simplify this by starting node with `cache_path` config?

https://github.com/foundry-rs/foundry/blob/9f11e6df38f6b38173eba0cc1134ea14413a6ce8/crates/anvil/src/config.rs#L967-L972

Reply:
The test actually verifies that the cache path is updated upon reset, the regex exists because [`cache_path`](https://github.com/foundry-rs/foundry-fork-db/blob/029852a61da1b26a49c9dfabf63493392ddd126e/src/cache.rs#L390) is private. If we expose that property we can get rid of the regex and check is straight away

Reply:
I see, makes sense. I think would be nicer to have a `JsonBlockCacheDB.get_cache_path`  exposed in fork-db rather than adding the new deps for testing, @yash-atreya @mattsse wdyt?

Reply:
I agree, I can make a PR for that there

Reply:
https://github.com/foundry-rs/foundry-fork-db/pull/42

Reply:
@nbaztec we released foundry-fork-db v0.11.1, please include in PR. thank you!

Reply:
sorry missed the ping here -.-